db/migrate/20160909161815_remove_intelli_sms_default.rb

```diff
@@ -0,0 +1,9 @@
+class RemoveIntelliSmsDefault < ActiveRecord::Migration
+  def up
+    Setting.where(default_outgoing_sms_adapter: "IntelliSms").update_all(default_outgoing_sms_adapter: nil)
```

Comment:
Please prefer raw SQL wherever possible in migrations. Relying on AR models can break if the models change in the future.